<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21" started-at="2016-07-05T13:34:32Z" finished-at="2016-07-05T13:34:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21" started-at="2016-07-05T13:34:32Z" finished-at="2016-07-05T13:34:32Z">
      <class name="com.apust.selenium.Login">
        <test-method status="FAIL" signature="init()[pri:0, instance:com.apust.selenium.Login@30c7da1e]" name="init" is-config="true" duration-ms="0" started-at="2016-07-05T16:34:32Z" finished-at="2016-07-05T16:34:32Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at java.util.Properties$LineReader.readLine(Properties.java:434)
	at java.util.Properties.load0(Properties.java:353)
	at java.util.Properties.load(Properties.java:341)
	at com.apust.selenium.util.PropertyLoader.loadProperty(PropertyLoader.java:20)
	at com.apust.selenium.pages.TestBase.init(TestBase.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="testLogin()[pri:0, instance:com.apust.selenium.Login@30c7da1e]" name="testLogin" duration-ms="1" started-at="2016-07-05T16:34:32Z" finished-at="2016-07-05T16:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method status="SKIP" signature="setScreenshot(org.testng.ITestResult)[pri:0, instance:com.apust.selenium.Login@30c7da1e]" name="setScreenshot" is-config="true" duration-ms="0" started-at="2016-07-05T16:34:32Z" finished-at="2016-07-05T16:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setScreenshot -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.apust.selenium.Login@30c7da1e]" name="tearDown" is-config="true" duration-ms="0" started-at="2016-07-05T16:34:32Z" finished-at="2016-07-05T16:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.apust.selenium.Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
